src/config.rs

```diff
@@ -763,15 +763,26 @@ pub struct Plugins {
     pub prewarmer: Option<Prewarmer>,
 }
 
+pub trait Plugin {
```

Comment:
Nice to have a trait, but don't think you need it for this particular fix. We should discuss what trait we should have for a Plugin to implement, so we can start having a real plugin system.

Reply:
Second thought I see why you added it here.

Reply:
I'm also thinking about the plugin system. I should write down my thoughts and share it with you. Please do the same if you have such notes.